packages/ppx_default/ppx_default.0.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Generate default values for your types"
+maintainer: ["franciscosantosfcp@hotmail.com"]
+authors: ["Francisco Santos"]
+license: "MIT"
+homepage: "https://github.com/ProgramingIsTheFuture/ppx_default"
+bug-reports: "https://github.com/ProgramingIsTheFuture/ppx_default/issues"
+depends: [
+  "dune" {>= "3.10"}
+  "ocaml" {>= "4.14"}
+  "ppxlib"
```

Comment:
lower bounds check failed for ocaml.4.14. seems like ppxlib needs a lower bound

- https://github.com/ocaml-ppx/ppxlib/blob/af1756cfcfb6ebabd2ebf2234109cd50fce3f2f8/CHANGES.md#0250-03032022

```suggestion
  "ppxlib" {>= "0.25"}
```

```
#=== ERROR while compiling ppx_default.0.1.1 ==================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/ProgramingIsTheFuture/ppx_default/releases/download/v0.1.1/ppx_default-v0.1.1.tbz)
# path                 ~/.opam/4.14/.opam-switch/build/ppx_default.0.1.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ppx_default -j 47 @install
# exit-code            1
# env-file             ~/.opam/log/ppx_default-7-568663.env
# output-file          ~/.opam/log/ppx_default-7-568663.out
### output ###
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/bytecomp/ocaml-compiler-libs.bytecomp.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/common/ocaml-compiler-libs.common.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/shadow/ocaml-compiler-libs.shadow.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/toplevel/ocaml-compiler-libs.toplevel.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I .ppx_default.objs/byte -I /home/opam/.opam/4.14/lib/ocaml-compiler-libs/common -I /home/opam/.opam/4.14/lib/ocaml-compiler-libs/shadow -I /home/opam/.opam/4.14/lib/ocaml/compiler-libs -I /home/opam/.opam/4.14/lib/ppx_derivers -I /home/opam/.opam/4.14/lib/ppxlib -I /home/opam/.opam/4.14/lib/ppxlib/ast -I /home/opam/.opam/4.14/lib/ppxlib/astlib -I /home/opam/.opam/4.14/lib/ppxlib/print_diff -I /home/opam/.opam/4.14/lib/ppxlib/stdppx -I /home/opam/.opam/4.14/lib/ppxlib/traverse_builtins -I /home/opam/.opam/4.14/lib/sexplib0 -I /home/opam/.opam/4.14/lib/stdlib-shims -no-alias-deps -o .ppx_default.objs/byte/ppx_default.cmo -c -impl ppx_default.ml)
# File "ppx_default.ml", line 158, characters 12-37:
# 158 |             Location.error_extensionf ~loc:ptype_loc
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound value Location.error_extensionf
```
```
The following actions will be performed:
=== downgrade 9 packages
  - downgrade cppo                1.6.9 to 1.6.1     [required by ppx_deriving]
  - downgrade dune                3.11.0 to 3.10.0   [required by ppx_default]
  - downgrade ocaml-compiler-libs v0.12.4 to v0.11.0 [uses jbuilder]
  - downgrade ocamlfind           1.9.6 to 1.9.1     [required by ppx_deriving]
  - downgrade ppx_derivers        1.2.1 to 1.0       [required by ppx_deriving]
  - downgrade ppx_deriving        5.2.1 to 5.2       [required by ppx_default]
  - downgrade ppxlib              0.31.0 to 0.24.0   [required by ppx_default]
  - downgrade sexplib0            v0.16.0 to v0.12.0 [uses dune]
  - downgrade stdlib-shims        0.3.0 to 0.1.0     [uses dune]
=== recompile 2 packages
  - recompile ppx_default         0.1.1 (pinned)
  - recompile result              1.5                [uses dune]
=== install 2 packages
  - install   base-bytes          base               [required by cppo]
  - install   jbuilder            1.0+beta20.1       [required by cppo]
```